modules/calib3d/include/opencv2/calib3d.hpp

```diff
@@ -574,6 +574,8 @@ projections, as well as the camera matrix and the distortion coefficients.
         - Thus, given some data D = np.array(...) where D.shape = (N,M), in order to use a subset of
         it as, e.g., imagePoints, one must effectively copy it into a new array: imagePoints =
         np.ascontiguousarray(D[:,:2]).reshape((N,1,2))
+   -   The methods **SOLVEPNP_DLS** and **SOLVEPNP_UPNP** cannot be used as the current implementations are
+       unstable and sometimes give completly wrong results.
```

Comment:
It's better to add that in case of passing `SOLVEPNP_DLS` or `SOLVEPNP_UPNP` flag, `SOLVEPNP_EPNP` will be used.